packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/DynamicZone/tests/ComponentCategory.test.tsx

```diff
@@ -37,21 +36,9 @@ describe('ComponentCategory', () => {
     expect(getByText(/myCategory/)).toBeInTheDocument();
   });
 
-  it('should call the onToggle callback when the accordion trigger is pressed', async () => {
```

Comment:
Why delete this test?

Reply:
It's testing an implementation detail, it doesn't really provide any value to us imho